tests/integration/targets/win_zip/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+win_zip_name: .ÅÑŚÌβŁÈ [$!@^&test(;)] 👋
```

Comment:
Using some non-ASCII characters here to test out some edge cases when it comes to Windows. This ensures that the module is able to handle these non-ASCII chars and verifies the UTF-8 encoding we specify is actually in affect.